### PR TITLE
Migration

### PR DESCRIPTION
**Pull request by yegonkimutai**
In this PR, I did the following ;
- [ ] Created tables as per the projects requirements.
- [ ] Save the tables to the database.

### CODE REVIEW

File: db/migrate/20230711161537_create_comments.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :user, null: false, foreign_key: { to_table: :users}
```
Comment (MahmoudBakr23) [inline db/migrate/20230711161537_create_comments.rb]: - [ ] Please modify the `user` attribute to `author` as you did in the posts table 🙏 This is according to the project requirements

File: db/migrate/20230711162014_create_likes.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: { to_table: :users}
```
Comment (MahmoudBakr23) [inline db/migrate/20230711162014_create_likes.rb]: - [ ] Please modify the `user` attribute to `author` as you did in the posts table 🙏 This is according to the project requirements

File: README.md [hunk 3]
```diff
@@ -1,24 +1,148 @@
...
+
+I would like to thank Microverse.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT] licensed.
```
Comment (MahmoudBakr23) [inline README.md]: - [ ] Please add a valid link to the MIT license for a completely professional readme file 🙏

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @yegonkimutai 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Well done generating the correct migrations so far
- Descriptive PR and commit messages
- Correct Linters

However, there is always room for improvement.

> I'd like to remind you that your long work hours are appreciated. However, the code review process is for improving your code, not criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

N/A

Cheers, and Happy coding!👏👨‍💻💻

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @yegonkimutai, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✔️
- Changes fixed ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**